test/rules/no-for-in/test.ts.lint

```diff
@@ -0,0 +1,23 @@
+// this should pass
```

Comment:
no need to write this, it's implied by the lack of lint failure marker

Reply:
```suggestion
```